disputer/disputer.js

```diff
@@ -71,37 +73,52 @@ class Disputer {
           at: "Disputer",
           message: "Cannot dispute liquidation: not enough collateral (or large enough approval) to initiate dispute.",
           id: disputeableLiquidation.id,
-          sponsor: disputeableLiquidation.sponsor
-        });
-
-        Logger.debug({
-          at: "Disputer",
-          message: "Dispute call error message",
-          id: disputeableLiquidation.id,
           sponsor: disputeableLiquidation.sponsor,
           error: error
         });
         continue;
       }
 
-      // TODO: handle transaction failures.
-      const receipt = await dispute.send({
-        from: this.account,
-        gas: 1500000,
-        gasPrice: this.gasEstimator.getCurrentFastPrice()
-      });
+      disputePromises.push(
+        dispute
+          .send({
+            from: this.account,
+            gas: 1500000,
+            gasPrice: this.gasEstimator.getCurrentFastPrice()
+          })
+          .catch(error => {
+            Logger.error({
+              at: "Disputer",
+              message: `Failed to dispute liquidation: ${error.message}`,
+              from: this.account,
+              gas: 1500000,
+              gasPrice: this.gasEstimator.getCurrentFastPrice(),
+              error
+            });
+          })
+      );
+    }
+
+    // Resolve all promises in parallel.
+    let promiseResponse = await Promise.all(disputePromises);
+
+    for (const response of promiseResponse) {
+      // response is undefined if an error is caught.
+      if (!response) {
```

Comment:
Needed to add this check here due to the `.catch` addition. If an error is caught, no response is sent